test/com/comeet/utilities/ExchangeUtility.java

```diff
@@ -0,0 +1,28 @@
+package com.comeet.utilities;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+import microsoft.exchange.webservices.data.core.ExchangeService;
+import microsoft.exchange.webservices.data.core.enumeration.misc.ExchangeVersion;
+import microsoft.exchange.webservices.data.credential.ExchangeCredentials;
+import microsoft.exchange.webservices.data.credential.WebCredentials;
+
+public class ExchangeUtility {
```

Comment:
We've got an interface for classes creating an Exchange Service: ExchangeServiceFactory